patches/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java.patch

```diff
@@ -1,5 +1,25 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java
+@@ -28,7 +28,7 @@
+     private static float field_74587_g;
+     private static float field_74596_h;
+ 
+-    public static void func_74583_a(EntityPlayer p_74583_0_, boolean p_74583_1_)
++    public static void updateRenderInfo(Entity entityIn, boolean p_74583_1_)
+     {
+         GlStateManager.func_179111_a(2982, field_178812_b);
+         GlStateManager.func_179111_a(2983, field_178813_c);
+@@ -38,8 +38,8 @@
+         GLU.gluUnProject(f, f1, 0.0F, field_178812_b, field_178813_c, field_178814_a, field_178810_d);
+         field_178811_e = new Vec3d((double)field_178810_d.get(0), (double)field_178810_d.get(1), (double)field_178810_d.get(2));
+         int i = p_74583_1_ ? 1 : 0;
+-        float f2 = p_74583_0_.field_70125_A;
+-        float f3 = p_74583_0_.field_70177_z;
++        float f2 = entityIn.field_70125_A;
++        float f3 = entityIn.field_70177_z;
```

Comment:
You should add
`net/minecraft/client/renderer/ActiveRenderInfo.updateRenderInfo(Lnet/minecraft/entity/Entity;Z)V=|p_74583_0_,p_74583_1_`
to forge.exc to avoid this patch